clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleCredentialsInitializer.groovy

```diff
@@ -35,6 +37,15 @@ import org.springframework.context.annotation.Scope
 @Configuration
 class GoogleCredentialsInitializer implements CredentialsInitializerSynchronizable {
 
+
+  @Autowired
+  GoogleExecutor google_executor  // Not used, just here to force initialization ordering
+
+  @Bean
```

Comment:
Rather than creating the `@Bean`/`@Autowired` pair here, can we add `@Component` and `@DependsOn` to specify the initialization ordering?

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/annotation/DependsOn.html

Reply:
I couldnt get this to work.
Made the other suggestions though.